Packs/Core/ReleaseNotes/3_0_84.md

```diff
@@ -0,0 +1,14 @@
+
+#### Playbooks
+
+##### New: Unsigned and unpopular process performed an injection
+
+- New: This playbook is designed to handle process injection alerts by analyzing preceding events, checking for agent prevention rules, and determining remediation actions. The playbook follows a logical flow to assess the risk, suggest improvements, and remediate potential threats.
```

Comment:
This playbook handles process injection alerts by analyzing preceding events, checking for agent prevention rules, and determining remediation actions. It follows a logical flow to assess the risk, suggest improvements, and remediate potential threats.